containers/flatpak/test/test-browser

```diff
@@ -83,5 +98,18 @@ class BrowserTest(unittest.TestCase):
         self.assertEqual(self.run_js("test-browser-login.js"), "page-load")
         self.assertEqual(self.run_js("test-browser-overview.js"), "PASS")
 
+    def testRecentHosts(self):
+        self.assertIn(self.run_js("test-browser-wait-init.js"), ["found", "page-load"])
+
+        # try to log in by clicking on `localhost`
+        self.assertEqual(self.run_js("test-browser-login-recent.js"), "page-load")
+        self.assertEqual(self.run_js("test-browser-logout.js"), "PASS")
```

Comment:
This is potentially racy -- it could be that the connection gets closed before it gets to `PASS`, so accepting "page-load" in addition here would make sense. But that's not what breaks the test.

Reply:
done in fixup